controllers/nugus_controller/nugus_controller.cpp

```diff
@@ -133,9 +133,25 @@ int main(int argc, char** argv) {
         return 0;
```

Comment:
Does returning non-zero here make sense since there was an error? (same with other similar returns below).

Reply:
What error code would you suggest

Reply:
`1`, or better yet, [`EXIT_FAILURE`](https://stackoverflow.com/a/13667390/5800506)